packages/iTwinUI-react/src/core/TimePicker/TimePicker.tsx

```diff
@@ -112,6 +161,28 @@ export type TimePickerProps = {
    * @default (meridiem: MeridiemType) => meridiem
    */
   meridiemRenderer?: (meridiem: MeridiemType) => React.ReactNode;
+  /**
+   * Use combined time renderer. Combines hour, minute, and seconds into one column.
+   * @default false
+   */
+  useCombinedRenderer?: boolean;
```

Comment:
Thanks for reminding me; added

Reply:
Virtualization is automatic for combined types.

Reply:
Ohh, that's good. I was confused because in other components we have to manually enable virtualization. E.g. `Table` has an `enableVirtualization` prop.

So would be good to document this as mentioned here (https://github.com/iTwin/iTwinUI-react/pull/844#discussion_r1049008729) to avoid confusion.

Reply:
Yeah, I did it this way because in the vast majority of cases I forsee virtualization being necessary. It's only not necessary if users only have hours (and why use combined renderer?) or very restricted minutes (eg 30 minute increments). I updated the description.